sdk-extensions/jfr-events/src/main/java/io/opentelemetry/sdk/extension/jfr/JfrSpanProcessor.java

```diff
@@ -26,11 +22,16 @@
  */
 public class JfrSpanProcessor implements SpanProcessor {
 
-  private volatile Map<SpanContext, SpanEvent> spanEvents =
-      new MapMaker().concurrencyLevel(16).initialCapacity(128).weakKeys().makeMap();
+  private final WeakConcurrentMap<SpanContext, SpanEvent> spanEvents =
+      new WeakConcurrentMap.WithInlinedExpunction<>();
```

Comment:
I ask you... Is "expunction" a word? really?

Reply:
Don't ask me ask raph :P